src/app/middleware/feathers-json.js

```diff
@@ -54,6 +54,16 @@ export default function (options) {
       delete files['server/middleware/body-parser'];
     }
 
+    template.plugins.push({
+      require: 'feathers-logger',
+      options: [{
+        require: 'bunyan',
+        options:[{
+          name: name
+        }]
+      }]
+    });
+
```

Comment:
This can be made configurable by prompts but I don't see the need to be honest. This is a really sane default and the more configurations we support in the wizard, the harder our lives are.